nixos/modules/services/networking/ddclient.nix

```diff
@@ -52,6 +52,16 @@ in
         '';
       };
 
+      configFile = mkOption {
+        default = "/etc/ddclient.conf";
+        type = str;
```

Comment:
I meant to only change the default to be a string; the `type` should probably still be `types.path` (same thing only requiring that the string looks like an absolute path).